upgrading/upgrading_roxctl/upgrade-from-40-43.adoc

```diff
@@ -0,0 +1,67 @@
+[id="upgrade-from-40-43"]
+= Upgrading from version 3.0.40 till 3.0.43
+include::modules/common-attributes.adoc[]
+:context: upgrade-from-40-43
+
+toc::[]
+
+[role="_abstract"]
+
+Upgrade to the latest version of Red Hat Advanced Cluster Security for Kubernetes from version 3.0.44 or higher.
```

Comment:
This sentence is confusing for a few reasons: 
-  I think "till" --> "through" as in the original docs makes more sense in the title. 
- The title says upgrading "from 3.0.40 till 3.0.43" and the intro sentence says "from 3.0.44". I think in this file that line 10 should say "from 3.0.40 through 3.0.43" to match the title.
- We don't generally precede our versions numbers with the word "version". ISG has some guidance about this:
  > At the first occurrence in text, it is preferable to spell out the full official name of the product and the version...In running text, it is acceptable to use the word version, lowercase, to make a generic reference to a product name that has already been mentioned.

  Thus, it's long-winded but I'd suggest revising the assembly titles to "Upgrading from Red Hat Advanced Cluster Security for Kubernetes 3.0.40 through 3.0.43" and "Upgrading Red Hat Advanced Cluster Security for Kubernetes 3.0.44 or higher" and changing dropping the word "version" when it precedes a number.